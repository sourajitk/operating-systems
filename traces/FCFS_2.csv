1,0,4
3,0,5
5,0,1
7,3,2
9,3,5
11,3,9
13,5,2
15,5,1
