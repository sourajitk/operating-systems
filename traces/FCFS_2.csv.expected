1,4
3,9
5,10
7,12
9,17
11,26
13,28
15,29
